packages/e2e-tests/src/specs/editor/inspector.js

```diff
@@ -24,7 +24,7 @@ describe('Inspector', () => {
       await createNewStory();
 
       await expect(page).toClick('li[role="tab"]', { text: 'Document' });
-      await await expect(page).toMatchElement('[aria-label="Publisher Logo"]');
```

Comment:
Ah I think this test was failing on mine too. Thanks for looking into it!

Reply:
not sure it's fixed but now it doesn't await twice 😂